packages/rancher-vsphere-cpi/package.yaml

```diff
@@ -1,2 +1,2 @@
 url: local
-packageVersion: 0
+packageVersion: 1
```

Comment:
Fairly sure this needs a rebase. If this is a local chart shouldn't it not use a packageVersion anymore @aiyengar2 / @jiaqiluo ? I know the "new" changes haven't gotten in yet, but for a local chart you would just bump the chart.yaml. 
@PennyScissors I forget how local this chart is, doesn't it exist upstream ? Are we in line with their versions right now ?

Reply:
@aiyengar2 Please correct me if this is not correct:
For the local chart, we should change the value of `version` in the chart.yaml file directly instead of the packageVersion. 
But it will not error out if we use the `packageVersion` as this PR does.

Reply:
That's correct @jiaqiluo.

@cbron iirc, vsphere is a local chart that technically has an upstream, similar to Istio, so maybe bumping the packageVersion is the right move here? If there is no upstream, we should just bump the patch version if possible since we want to ideally deprecate the packageVersion anyways.

Reply:
Is the upstream versioned or just a bunch of manifests @PennyScissors ? If versioned then we should use 100.0.0. If not we can do whatever we want with it, if 2.5 was on 1.x.x then 2.6 would be on 2.x.x.

Reply:
Yes, upstream is just a bunch of manifests.